ert/experiment_server/_state_machine.py

```diff
@@ -0,0 +1,23 @@
+from collections import defaultdict
+from typing import Dict, List
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class StateMachine:
```

Comment:
Could we be more explicit and call this `ExperimentStateMachine`, so that it is clear that it is the state of the Experiment that is altered? I guess the probability is not zero that we in the future might have another stateful system in need of a state machine.